### PR TITLE
S01 us08 creation of form menu

### PR DESCRIPTION
**Pull request by Thomas-Lunardo**
I add two buttons on the signup.html.twig.
The javascript code allows to hide the useless form.

### CODE REVIEW

File: src/View/Home/signup.html.twig [hunk 1]
```diff
@@ -3,63 +3,79 @@
...
+					</fieldset>
+					<div class="buttons-line">
+						<button type="submit">Connexion</button>
+					</div>
+				</form>
+		</div>
 	</div>
 {% endblock %}
+
+{% block javascript %}
```
Comment (benrcd) [inline src/View/Home/signup.html.twig]: Javascript goes to javascript file.

File: src/View/Home/signup.html.twig [hunk 2]
```diff
@@ -3,63 +3,79 @@
...
-			<form action="" method="post">
-				<h4>Espace d'inscription hôte</h4>
-				<fieldset>
-					<label for="host-name">Votre identifiant :
+		<div class="choose-form">
+			<button id="btn1">Hôte</button>
+			<button id="btn2">Musicien</button>
+		</div>
+		<div id="form-container">
+				<form action="" method="post" id="form1" style="display: none;">
```
Comment (benrcd) [inline src/View/Home/signup.html.twig]: style="display: none;
This should be in your css file, not here (same for the other form)

**Review by benrcd — CHANGES_REQUESTED** [review]

File: src/View/Home/signup.html.twig (hunk 1 above)
Comment (Thomas-Lunardo) [reply src/View/Home/signup.html.twig]: done

File: src/View/Home/signup.html.twig (hunk 2 above)
Comment (Thomas-Lunardo) [reply src/View/Home/signup.html.twig]: done

**Review by victorzanotelli — APPROVED** [review]

**Review by victorzanotelli — APPROVED** [review]

**Review by benrcd — APPROVED** [review]

**Pull request merged**